planetary-social/nos
# Enable Phase 1 of new media display by removing the feature flag

## Issues covered
#1538 

## Description
Enables Phase 1 of the new media display for everyone by removing the feature flag. We'll add a new feature flag when the time comes.

## How to test
1. Navigate to the Feed
2. Scroll until you find a post with multiple photos or videos or links
3. Observe the orange dot, confirming that the new media display is enabled
4. Navigate to Settings
5. Verify the Enable new media display toggle is gone

## Screenshots/Video
| Feed | Settings |
|--------|--------|
| ![Simulator Screenshot - iPhone 15 Pro - 2024-10-02 at 15 44 13](https://github.com/user-attachments/assets/63eb646a-828c-4ae7-ba36-fdbc626011ca) | ![Simulator Screenshot - iPhone 15 Pro - 2024-10-02 at 15 43 26](https://github.com/user-attachments/assets/9fc42f27-cab6-463d-af97-cb1527c82c57) |
